src/components/ec-checkbox/ec-checkbox.story.js

```diff
@@ -5,26 +5,26 @@ import EcCheckbox from './ec-checkbox.vue';
 const stories = storiesOf('Checkbox', module);
 
 stories
-  .add('basic', () => ({
+  .add('all', () => ({
     components: { EcCheckbox },
     props: {
       valueFromPropsChecked1: {
-        default: boolean('Checkbox 1'),
+        default: boolean('"Label and Error messages coming from props" Checkbox'),
```

Comment:
I found this labels very confusing, you can make it better by using knob groups: https://www.npmjs.com/package/@storybook/addon-knobs#categorization.

Reply:
Done